blog/en/blog/2023/11/23/bi-weekly-report.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Biweekly Report (November 06 - November 19)"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: Our bi-weekly Apache APISIX community report is your window into the project's weekly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/11/20/LQgkAgUZ_11.20-Eng.png
+---
+
+> We have recently made some fixes and improvements to specific features within Apache APISIX. The update includes supporting Nacos AK/SK authentication. For additional information, please consult the bi-weekly report.
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'Many hands make light work' rings true in our way, made possible by the collective dedication of our community.
+
+From 11.06 to 11.19, a total of 15 contributors made 26 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+We have recently addressed and enhanced various features, and here is a summary of the update:
+
+1. Support Nacos AK/SK authentication
+
+Our bi-weekly Apache APISIX community report is your window into the project's weekly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/11/20/j4c7LdeJ_11.20-Con.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/11/20/eaimWsfQ_11.20-New.png)
+
+## Highlights of Recent Features
+
+### Apache APISIX
+
+- [Support Nacos AK/SK authentication](https://github.com/apache/apisix/pull/10445) (Contributor: [yuweizzz](https://github.com/yuweizzz))
+
```

Comment:
```suggestion
## Highlight of Recent Feature

- [Support Nacos AK/SK authentication](https://github.com/apache/apisix/pull/10445) (Contributor: [yuweizzz](https://github.com/yuweizzz))

```

Reply:
Similar reason, when only 1 change in 1 product, you can be more flexible with the template.